pkg/api/client.go

```diff
@@ -38,6 +38,15 @@ type ClientOptions struct {
 	// Default is no logging.
 	Log io.Writer
 
+	// LogIgnoreEnv disables respecting the GH_DEBUG environment variable for logging.
+	LogIgnoreEnv bool
+
+	// LogColorize enables colorized logging to Log for display in a terminal.
```

Comment:
```suggestion
	// LogColorize enables colorized logging to Log for display in a terminal.
	// Default is no coloring.
```

With the comments I have been trying to describe the default state for each options as I generally find it useful when I look at documentation.